Source/WebCore/css/makevalues.pl

```diff
@@ -193,31 +187,40 @@
     $maxLen = length($name);
   }
 }
-my $num = $i - $first;
 my $last = $i - 1;
 
 print HEADER "};\n\n";
-print HEADER "const int firstCSSValueKeyword = $first;\n";
-print HEADER "const int numCSSValueKeywords = $num;\n";
-print HEADER "const int lastCSSValueKeyword = $last;\n";
-print HEADER "const size_t maxCSSValueKeywordLength = " . $maxLen . ";\n";
-print HEADER << "EOF";
 
-ASCIILiteral getValueName(unsigned short id);
-const AtomString& getValueNameAtomString(CSSValueID id);
-String getValueNameString(CSSValueID id);
+print HEADER "constexpr uint16_t numCSSValueKeywords = $i;\n";
+print HEADER "constexpr uint16_t lastCSSValueKeyword = $last;\n";
+print HEADER "constexpr unsigned maxCSSValueKeywordLength = $maxLen;\n";
 
-inline CSSValueID convertToCSSValueID(int value)
-{
-    ASSERT(value >= firstCSSValueKeyword && value <= lastCSSValueKeyword);
-    return static_cast<CSSValueID>(value);
-}
+print HEADER << "EOF";
+
+CSSValueID findCSSValueKeyword(const char* characters, unsigned length);
+ASCIILiteral nameLiteral(CSSValueID);
+const AtomString& nameString(CSSValueID);
+
+struct AllCSSValueKeywordsRange {
+    struct Iterator {
+        uint16_t index { 0 };
+        constexpr CSSValueID operator*() const { return static_cast<CSSValueID>(index); }
+        constexpr Iterator& operator++() { ++index; return *this; }
+        constexpr bool operator!=(std::nullptr_t) const { return index < numCSSValueKeywords; }
+    };
+    static constexpr Iterator begin() { return { }; }
```

Comment:
Same thought about a `size()` function, but this one could just return `numCSSValueKeywords`